saleor/order/models.py

```diff
@@ -500,3 +500,10 @@ class Meta:
 
     def __repr__(self):
         return f"{self.__class__.__name__}(type={self.type!r}, user={self.user!r})"
+
+
+class Invoice(models.Model):
+    order = models.ForeignKey(Order, null=True, on_delete=models.SET_NULL)
+    number = models.CharField(max_length=64)
```

Comment:
Are you sure that 64 is enough in any cases?

Reply:
in Poland in my entire life i didn't see a number larger than 20. But since this is a multi country project we can bump it up to 128. That should be more than safe since i googled a bit and couldn't find informations about extra long invoice numbers, in countries i've seen it's kept short.

Reply:
I mean, I believe `999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999` (64) orders is enough.